### PR TITLE
Update ready.md

### PR DESCRIPTION
**Pull request by bsantacruzms**
- Adding CycleCloud features
- Complementing information with links.
  

  
<details><summary>AFTER YOUR PR HAS BEEN CREATED, expand this section for tips and additional instructions.</summary>    
      

These are common guidelines for contributions across the repos managed by the Cloud Architecture Content Team (CACT). Some repositories may have additional specific requirements that are not listed here.   

## Guidance for all contributors  
  
  | **Topic** | **Guidance** |
  | ----------| ------------ |
  | **Draft PR** | If your PR will be a work-in-progress for more than a day or two, select the **Convert to draft** link in the upper right of the page (under **Reviewers**) to change it to a draft. For future reference, you can also do this using the **Create pull request** button drop-down during PR creation. | 
  | **ms.date metadata** | <ul><li>Don't update an article's "ms.date" metadata property unless you've done a **full freshness review** of the content. A full freshness review includes changes required to correct or improve the **full** technical accuracy of the article.</li><li>Don't update "ms.date" if you're doing targeted changes to improve non-technical aspects of the article, such as editorial quality, art improvements, article template alignment, etc.</li><li>If you've changed any "ms.date" properties for work that wasn't part of full review for freshness, please reset them to their previous value.</li></ul> | 
  | **Placement and linking** | If you're creating a new article or articles, include updates to the related TOC.yml file to propose where the article(s) should be placed. Also consider other places within the document set where it would be beneficial to cross-reference and link to your new article(s). | 
  | **PR build** | After you open your PR, and for each successive commit that you push to your branch, the publishing platform will run validation on the files in your pull request. A summary of the build results for each file will be inserted inline into your pull request, which includes any build suggestions/warnings/errors. PRs cannot be merged until all build errors and most warnings are resolved. |
  | **Publishing** | Following a successful merge, most repos publish to the live site at least once per (business) day, usually around 10am Pacific. |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li></ul>
  
## Additional guidance for private repos and internal contributors  

  | **Topic** | **Guidance** | 
  | ----------| ------------ | 
  | **PR size** | If your PR is more than ~5 lines of changes, or you'd like for the changes to go through editorial or larger review, open a contribution request at https://aka.ms/Contribution and include a link to the PR in response #8. Once it's processed, you'll be notified of the next steps.  |
  | **PR title prefix** | Select the **Edit** button to the right of the PR title if you need to revise it. The following prefixes are reserved for specific contribution types:<br/><br/><ul><li>**[Quality Check]** - maintenance work related to content quality (edit passes, art improvements, template alignment)</li><li>**[LinkFix]** - recurring/adhoc PRs to correct link URLs</li><li>**[Pipeline]** - new/updated contributor success pipeline content</li><li>**[WIP]** - a work-in-progress draft requiring several days/weeks</li></ul> |
  | **PR preview** | Following successful build of your PR, publishable files will also include **Preview URL** links to staged previews of your new/updated articles. Be sure to review these for verification of your intended contributions, or to send to other internal contributors for review. |
  | **PR sign-off (public repo)** | If an article you own is updated in a public repo PR, you are responsible for sign-off. You will be automatically notified via email. The PR will not be merged until you've had a chance to review and sign-off. |
  | **PR sign-off (private repo)** | After you've completed your proposed changes, addressed build warnings, and completed all review work, you can begin the sign-off process for review and merge:<br/><br/><ol><li>If your PR is in draft mode, remove "[WIP]" from the title and select **Ready for review** button at the bottom of the PR.</li><li>Enter "#sign-off" in a new comment. This comment indicates that **you're confident the work meets or exceeds Microsoft's standards for publication**, and will trigger the review process.</li><li>Your PR may be selected for initial review by the CACT. Following CACT review, you may receive questions or requests for additional changes. You should have initial feedback from CACT review within a few business days. If you have an urgent request or need to contact the team, please mention `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers` in your PR and someone will get back to you. After CACT review is complete, a `CACT #sign-off` will be added.</li><li>Final review/merge is done by the PR review team. The PR team may also respond with feedback, categorized as "Blocking" (requires action from you), or "Non-blocking" (to be addressed in a future PR).</li></ol> |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com internal contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li><li>Authoring templates: [architecture-center-pr](https://review.learn.microsoft.com/help/contribute/architecture-center/templates/sample-solution-templates?branch=main), [well-architected-pr](https://review.learn.microsoft.com/help/contribute/global-waf-template?branch=main)</li><li>To contact the CACT use [e-mail](mailto:cact-pr-reviewers@microsoft.com?subject=Help%20with%20pull%20request), or @mention our GitHub team in your PR comments using: `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers`</li></ul>
</details>

### CODE REVIEW

**Comment by Jak-MS** [comment]
@Rajani-Janaki-Ram 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/scenarios/azure-hpc/ready.md [hunk 1]
```diff
@@ -34,42 +34,52 @@ As you can see from these questions, the deployment model varies depending on yo
...
 
-**Azure Batch** and **Azure Cycle Cloud** are both Azure provided 1st party service while **Azure HPC OnDemand Platform (AzHop)** is an automation built on top of Azure Cycle Cloud which delivers an end-to-end deployment mechanism for a complete HPC cluster solution in Azure.
+[Azure Batch](/azure/batch/batch-technical-overview) and [Azure CycleCloud](/azure/cyclecloud/overview) are both Azure provided 1st party service while **Azure HPC OnDemand Platform** ([AzHOP](https://azure.github.io/az-hop/)) is an automation built on top of Azure Cycle Cloud which delivers an end-to-end deployment mechanism for a complete HPC cluster solution in Azure.
 
-### Batch
+### Azure Batch
 
 - Batch is designed to run large-scale parallel and HPC batch jobs efficiently in Azure.
-- Batch creates and manages a pool of compute nodes, or virtual machines (VMs). You can also use Batch to install the applications that you want to run and to schedule jobs to run on the nodes.
+- Batch creates and manages a pool of computer nodes, or virtual machines (VMs). You can also use Batch to install the applications that you want to run and to schedule jobs to run on the nodes.
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:42]:
```suggestion
- Batch creates and manages a pool of compute nodes, or virtual machines (VMs). You can also use Batch to install the applications that you want to run and to schedule jobs to run on the nodes.
```

File: docs/scenarios/azure-hpc/ready.md [hunk 2]
```diff
@@ -34,42 +34,52 @@ As you can see from these questions, the deployment model varies depending on yo
...
 - There's no cluster or job scheduler software to install, manage, or scale.
 - Batch works well with intrinsically parallel workloads. These workloads have applications that can run independently, with each instance completing part of the work.
 - You can also use Batch to run tightly coupled workloads, where the applications that you run need to communicate with each other rather than running independently.
 - Batch tasks can run directly on VMs, or nodes, in a Batch pool. But you can also set up a Batch pool to run tasks in Docker-compatible containers on the nodes.
 
+
+
+### Azure CycleCloud
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:50]:
```suggestion
### Azure CycleCloud

```

File: docs/scenarios/azure-hpc/ready.md [hunk 3]
```diff
@@ -34,42 +34,52 @@ As you can see from these questions, the deployment model varies depending on yo
...
+- It supports various HPC schedulers, such as SLURM, OpenPBS, PBSPro, LSF, Grid Engine, and HTCondor12.
+- It allows you to provision infrastructure for HPC systems, such as virtual machines, scale sets, network interfaces, and disks.
+- It automatically scales the infrastructure to run jobs efficiently at any scale based on job load, availability, and time requirements.
+- It provides a rich, declarative, templating format to construct complete HPC environments on Azure.
+- It integrates with Azure services such as Azure Monitor and Azure Cost Management tools.
+
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:56]:
```suggestion
```

File: docs/scenarios/azure-hpc/ready.md [hunk 4]
```diff
@@ -34,42 +34,52 @@ As you can see from these questions, the deployment model varies depending on yo
...
+- AzHop provides an end-to-end deployment mechanism for a base HPC infrastructure on Azure utilizing CycleCloud to orchestrate jobs.
 - It delivers a complete HPC cluster solution that's ready for users to run applications on and that's easy for HPC administrators to deploy and manage.
-- AzHop uses various Azure building blocks. You can use it as is, or you can easily customize and extend it to meet any unmet requirements.
-- It includes an Open OnDemand portal for unified user access, remote shell access, remote visualization access, job submission, file access, and more.
-- It uses Active Directory for user authentication and domain control.
-- It uses OpenPBS or Simple Linux Utility for Resource Management (SLURM) as a job scheduler.
-- Dynamic resource provisioning and autoscaling are done by Azure CycleCloud preconfigured job queues and integrated health checks to quickly avoid nonoptimal nodes.
-- Azure NetApp Files delivers a shared file system for the home directory and applications.
+- AzHop uses various applications built-in that you can use “as is”, or you can easily customize and extend to meet any unmet requirements.
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:62]:
```suggestion
- AzHop uses various applications built-in that you can use "as is," or you can easily customize and extend to meet any unmet requirements.
```

File: docs/scenarios/azure-hpc/ready.md [hunk 5]
```diff
@@ -34,42 +34,52 @@ As you can see from these questions, the deployment model varies depending on yo
...
+- It includes an **Open OnDemand** portal for unified user access, remote shell access, remote visualization access, job submission, file access, and more.
+- It uses **Active Directory** for user authentication and domain control.
+- It uses **OpenPBS** or Simple Linux Utility for Resource Management (**SLURM**) as a job scheduler.
+- Dynamic resource provisioning and autoscaling are done by **CycleCloud** preconfigured job queues and integrated health checks to quickly avoid nonoptimal nodes.
+- **Azure NetApp Files** delivers a shared file system for the home directory and applications.
 
 ### Comparison chart
 
-|Feature | Batch | Azure CycleCloud  |
+|Feature | Azure Batch | CycleCloud  |
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:71]:
```suggestion
|Feature | Azure Batch | Azure CycleCloud |
```

File: docs/scenarios/azure-hpc/ready.md [hunk 6]
```diff
@@ -78,6 +88,6 @@ In contrast, Batch is mostly aimed at developers and teams who build a capabilit
 In summary, use Azure CycleCloud when you want to deploy an HPC environment with a specific scheduler in mind and need a complete HPC environment. Use Batch when you develop a product or service that requires large-scale parallel processing and you don't want to manage a workload scheduler.
 
 ## Next steps
-
+- Review available [High performance computing VM sizes](/azure/virtual-machines/sizes-hpc)
```
Comment (ckittel) [inline docs/scenarios/azure-hpc/ready.md:91]:
```suggestion

- Review available [High performance computing VM sizes](/azure/virtual-machines/sizes-hpc).
```

**Review by ckittel — COMMENTED** [review]
Minor edits

**Review by ckittel — APPROVED** [review]

**Comment by ckittel** [comment]
@bsantacruzms As a MSFT employee, in the future, please use `cloud-adoption-framework-pr` for your PRs.

CACT #sign-off

#remove-label:"cact-review-in-progress"
#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Comment by bsantacruzms** [comment]
> @bsantacruzms As a MSFT employee, in the future, please use `cloud-adoption-framework-pr` for your PRs.
> 
> CACT #sign-off
> 
> #remove-label:"cact-review-in-progress" #remove-label:"do-not-merge" #label:"ready-to-merge"

Sure, thank you for letting me know.

**Pull request merged**